pages/domains/index.tsx

```diff
@@ -34,39 +34,39 @@ export default function RegisterNamePage() {
   return (
     <div className="w-full text-black px-5 flex h-full flex-col items-center justify-start">
       <div className="w-full flex-col gap-8 py-10 flex justify-start max-w-[1216px]">
-        <Skeleton className="!w-full min-h-[500px]" loading={isLoading}>
-          {!address && !isLoading ? (
-            <div className="w-full flex flex-col gap-4 items-center justify-center">
-              <Heading level="2" className="text-start text-[26px]">
-                You have to connect your wallet to see your domains 😉
-              </Heading>
-              <div>
-                <ConnectButton.Custom>
-                  {({ openConnectModal }) => {
-                    return (
-                      <Button
-                        onClick={(e: any) => {
-                          openConnectModal();
-                          e.preventDefault();
-                        }}
-                        size="small"
-                        colorStyle="blueSecondary"
-                        prefix={<WalletSVG />}
-                      >
-                        Connect
-                      </Button>
-                    );
-                  }}
-                </ConnectButton.Custom>
-              </div>
+        {!address ? (
+          <div className="w-full flex flex-col gap-4 items-center justify-center">
+            <Heading level="2" className="text-start text-[26px]">
+              You have to connect your wallet to see your domains 😉
+            </Heading>
+            <div>
+              <ConnectButton.Custom>
```

Comment:
We are not anymore using ConnectButton, please search for ConnectWallet component or something like, we are using a button to connect Metamask only